src/components/TranslationChartImage.tsx

```diff
@@ -1,39 +1,19 @@
-import React from "react"
 import { useColorModeValue } from "@chakra-ui/react"
-import { useStaticQuery, graphql } from "gatsby"
-import GatsbyImage from "./GatsbyImage"
-import { getImage } from "../utils/image"
+import { Image } from "@/components/Image"
+import pageviewsLight from "@/public/translation-program/pageviews-light.png"
```

Comment:
```suggestion
import pageviewsLight from "./translation-program/pageviews-light.png"
```

Reply:
@nhsz Are you sure about this?

<img width="485" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/883de679-c4ac-4a7a-867a-313cfeba3066">

I get an error when I attempt this... pretty sure relative paths with module imports would reference from the directory of that file, not `public`.